meta.yaml

```diff
@@ -7,7 +7,9 @@ meta:
   include:
     - version.yaml
     - dobi.yaml
-    - yocto/dobi.yaml
+    - yocto/.generated_cpu01-bringup.yaml
```

Comment:
maybe it would be better to have one generated file which includes other generated?!

Reply:
generated files should be in a gen folder or similar to make cleanup easier

Reply:
What's the difference: rm -f "yocto/.generated*" to "rm -rf yocto/gen"?

Reply:
> 
> 
> maybe it would be better to have one generated file which includes other generated?!

Yes, but I don't have an easy way to do it. There is yocto_dobi_gen.yaml which generates the yocto/.generated* files, but I don't see how I can generate a dobi.yaml with the included files from it

Reply:
Generate only one dobi.yaml with all 3 variations